gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -144,7 +145,12 @@ public GobblinHelixJobLauncher(Properties jobProps, final HelixManager helixMana
 
     jobConfig = ConfigUtils.propertiesToConfig(jobProps);
 
-    this.stateStores = new StateStores(jobConfig, appWorkDir,
+    Config stateStoreJobConfig = ConfigUtils.propertiesToConfig(jobProps)
+        .withValue(ConfigurationKeys.STATE_STORE_FS_URI_KEY, ConfigValueFactory.fromAnyRef(
```

Comment:
It wasn't passed before and I don't need it, so I'm only making an incremental improvement that fixes my issue.